go.mod

```diff
@@ -60,8 +60,10 @@ require (
 	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
 	golang.org/x/net v0.0.0-20200707034311-ab3426394381
 	golang.org/x/sync v0.0.0-20200317015054-43a5402ce75a
-	golang.org/x/sys v0.0.0-20200728102440-3e129f6d46b1
+	golang.org/x/sys v0.0.0-20200810151505-1b9f1253b3ed
 	k8s.io/api v0.0.0-20190620084959-7cf5895f2711
 	k8s.io/apimachinery v0.19.0
 	k8s.io/client-go v0.0.0-20190620085101-78d2af792bab
 )
+
+replace github.com/containers/image/v5 => github.com/containers/image/v5 v5.5.2-0.20200902171422-1c313b2d23e0
```

Comment:
Isn't this a WIP until this is released?  I think we should release containers/image so that @mheon can release pdoman 2.1.0

Reply:
@mheon acked above (see https://github.com/containers/podman/pull/6811#issuecomment-688523677). We just need to release before v2.1.0 which is still a bit in the future.

Reply:
@rhatdan We're still waiting on a few more PRs in Podman land (I don't want to cut a 2.1.0 without https://github.com/containers/podman/pull/7452 because that's blocking some folks from moving off 1.9) so we have a bit of time.